components/ai_chat/core/common/mojom/ai_chat.mojom

```diff
@@ -198,12 +200,36 @@ struct UploadedFile {
   UploadedFileType type;
 };
 
+struct ImageContentBlock {
+  // Only base64 encoded data uri is supported
+  string image_url;
```

Comment:
just checking but we don't want this to be a `mojom.Url`? I know its a bit more of a pain to use but it is more explicit